dev-python/wurlitzer/wurlitzer-2.0.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Capture C-level stdout/stderr in Python "
+HOMEPAGE="https://github.com/minrk/wurlitzer"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
```

Comment:
This package has tests.

IUSE="test"                                                                     
RESTRICT="!test? ( test )"                                                      
                                                                                
BDEPEND="test? ( dev-python/pytest${PYTHON_USEDEP} )"

python_test() {                                                                 
    pytest -vv test.py || die "Tests fail with ${EPYTHON}"                      
}

Reply:
`distutils_enable_tests pytest` should do the same, right?

Reply:
I removed the tests again, I can't get them to work, it keeps telling me that no tests were run.